certbot-dns-linode/setup.py

```diff
@@ -0,0 +1,65 @@
+import sys
+
+from setuptools import setup
+from setuptools import find_packages
+
+version = '0.24.0.dev0'
+
+# Please update tox.ini when modifying dependency version requirements
```

Comment:
It looks like cc344bfd1 changed the equivalent comment in other `setup.py` files:

```
# Remember to update local-oldest-requirements.txt when changing the minimum
# acme/certbot version.
```

It looks like that same commit introduced a `local-oldest-requirements.txt` file in the root of each plugin. I'm not familiar with that change, but I assume you'll need to add one.